yjb-game/template-game/styles/styles.css

```diff
@@ -0,0 +1,153 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Mono&display=swap');
+
+::BaseStylesheet [stylesheet] html {
```

Comment:
I actually made a template game so that we could see how the file structure looks when we add a custom CSS file so this folder is technically not necessary in the long run. we can leave it in for now and restructure later. sorry for not mentioning this!